files/apt.sh

```diff
@@ -1,5 +1,3 @@
-#!/bin/bash
```

Comment:
instead of removing the shebang, should we add the executeable bit?

Reply:
These files (`files/*.sh`) only declare functions (with the exception of `common.sh` which also adjust environment variables), so if they had the x bit and a shebang they would still do nothing.  That's the reason I removed the shebang: they are not "entry points", only "support code".